docs/src/content/docs/tutorials/marketplace.mdx

```diff
@@ -0,0 +1,237 @@
+
+In the split-payments use case, there are three parties to the transaction:
+
+- Buyer: the purchaser of a good or service on the marketplace
+- Platform: the operator of the marketplace
+- Merchant: the seller of a good or service on the marketplace
+
+You will assume the role of the platform operator in this tutorial.
+
+## Dependencies
+
+<Dependencies />
+
+## Endpoints
+
+- <LinkOut href='https://openpayments.dev/apis/auth-server/operations/post-request/'>
+    Grant Request
+  </LinkOut>
+- <LinkOut href='https://openpayments.dev/apis/wallet-address-server/operations/get-wallet-address/'>
+    Get a Wallet address
+  </LinkOut>
+- <LinkOut href='https://openpayments.dev/apis/resource-server/operations/create-incoming-payment/'>
+    Create an Incoming Payment
+  </LinkOut>
+- <LinkOut href='https://openpayments.dev/apis/resource-server/operations/create-outgoing-payment/'>
+    Create an Outgoing Payment
+  </LinkOut>
+
+<Tabs>
+<TabItem label='TypeScript'>
+<Ts />
+
+## Steps
+
+### 1. Request an Incoming Payment grant for the merchant
+
+Request an `incomingPayment` grant from the merchant's wallet's AS.
+
+```ts
+const grant = await client.grant.request(
+  {
+    url: merchantWalletAddress.authServer
+  },
+  {
+    access_token: {
+      access: [
+        {
+          type: 'incoming-payment',
+          actions: ['list', 'read', 'read-all', 'complete', 'create']
+        }
+      ]
+    }
+  }
+)
+```
+
+### 2. Request an Incoming Payment grant for your platform
+
+Request an `incomingPayment` grant from your wallet's AS.
+
+```ts
+const grant = await client.grant.request(
+  {
+    url: platformWalletAddress.authServer
+  },
+  {
+    access_token: {
+      access: [
+        {
+          type: 'incoming-payment',
+          actions: ['list', 'read', 'read-all', 'complete', 'create']
+        }
+      ]
+    }
+  }
+)
+```
+
+### 3. Create an Incoming Payment for the merchant
+
+Create an `incomingPayment` on the merchant's wallet's Resource Server (RS) using the access token returned by the AS in the grant request in step 1 and specify the `merchantWalletAddress` with the URL of the merchant's wallet.
+
+Add the `incomingAmount` object and define the following properties:
+
+- `value` : the maximum allowable amount that will be paid to the merchant's wallet address.
+- `assetCode` : the ISO 4217 currency code representing the underlying asset used to make the payment.
+- `assetScale` : the scale of amounts denoted in the corresponding asset code.
+
+```ts
+const merchantIncomingPayment = await client.incomingPayment.create(
+  {
+    url: new URL(WALLET_ADDRESS).origin,
+    accessToken: INCOMING_PAYMENT_ACCESS_TOKEN
+  },
+  {
+    merchantWalletAddress: WALLET_ADDRESS,
+    incomingAmount: {
+      value: '1000',
```

Comment:
maybe it would be helpful to use specific amounts for this example (something like 950 for the merchant, and then 50 for the platform)